cmd/akamai/command.go

```diff
@@ -31,6 +31,10 @@ var (
 	gitProtocolFlag          string
 	gitopsTemplateURLFlag    string
 	gitopsTemplateBranchFlag string
+	gitopsRepoName			 string
```

Comment:
Can you run the `go fmt ./...` command so that the formatting is correctly applied to this change please? You only need to change files/lines you've updated.